sycl/include/sycl/ext/oneapi/experimental/group_helpers_sorters.hpp

```diff
@@ -102,14 +114,180 @@ template <typename Compare = std::less<>> class default_sorter {
                                           size_t range_size) {
     return range_size * sizeof(T) + alignof(T);
   }
+};
 
-  template <typename T, int dim = 1>
-  static constexpr size_t memory_required(sycl::memory_scope scope,
-                                          sycl::range<dim> r) {
-    return 2 * memory_required<T>(scope, r.size());
+template <typename T, std::size_t ElementsPerWorkItem = 1,
+          typename CompareT = std::less<>>
```

Comment:
Please, rearrange arguments according to the Spec: 
```cpp
    template<typename T,
             typename Compare = std::less<>,
             std::size_t ElementsPerWorkItem = 1>
    class group_sorter;
```

Reply:
Changed.